src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -27,6 +27,75 @@ public partial class UIElement : DependencyObject
 		internal const string DefaultHtmlTag = "div";
 
 		private readonly GCHandle _gcHandle;
+		private bool _isCssStyleUpdatesSuspended = false;
+
+		/// <summary>
+		/// Since there is no way to apply all set and reset updates in bulk, we create them of groups and execute
+		/// each group separately to maintain the desired order.
+		/// </summary>
+		private readonly Queue<IQueuedUpdatesGroup> _queuedUpdates = new Queue<IQueuedUpdatesGroup>();
```

Comment:
To maintain the order, this may still lead to multiple JS calls when SetStyles are mixed with ResetStyles, but the number of calls should be less than before.

In the future, we may try to add a mechanism that allows us to know whether the order is important or not and optimize this further.